src/layout/page-layout.vue

```diff
@@ -16,8 +16,13 @@
 <script lang="ts" setup>
   import { computed } from 'vue';
   import { useTabBarStore } from '@/store';
+  import { useRoute } from 'vue-router';
 
   const tabBarStore = useTabBarStore();
+  const useRoure = useRoute();
+  const key = computed(() => {
+    return useRoure.path + Math.random();
```

Comment:
Use random numbers? Isn't the path unique?

Reply:
This is to force a component refresh every time you switch routes